contracts/interfaces/stealth/IStealthTx.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.4;
+
+interface IStealthTx {
+  event StealthVaultSet(address _stealthVault);
+  event PenaltySet(uint256 _penalty);
+  event MigratedStealthVault(address _migratedTo);
+
+  function migrateStealthVault() external;
```

Comment:
```suggestion
```
might want to remove this is migration logic was deprecated

Reply:
It's getting rid of in my other branch 😊